processor/src/main/java/org/mapstruct/ap/model/source/builtin/BuiltInMappingMethods.java

```diff
@@ -35,7 +36,6 @@
 
     public BuiltInMappingMethods(TypeFactory typeFactory) {
         builtInMethods = Collections.newArrayList(
-            new JaxbElemToValue( typeFactory ),
             new DateToXmlGregorianCalendar( typeFactory ),
             new XmlGregorianCalendarToDate( typeFactory ),
```

Comment:
Processing succeeds with it and it appears to be in the runtime.

http://developer.android.com/reference/javax/xml/datatype/XMLGregorianCalendar.html